octoprint_mrbeam/translations/de/LC_MESSAGES/messages.po

```diff
@@ -2632,7 +2650,7 @@ msgstr "Fokussiere den Laserkopf, um ein gutes Ergebnis zu erzielen!"
 #: octoprint_mrbeam/templates/conversion_dialog.jinja2:708
 #: octoprint_mrbeam/templates/conversion_dialog.jinja2:711
 msgid " Learn how to do this."
-msgstr "Lerne wie das geht."
+msgstr "Lerne wie das geht"
```

Comment:
why you removed the "." ??

Reply:
But this warning is about the space and not the dot

Reply:
Fixed